lib/date_time/date_time.c

```diff
@@ -710,15 +367,6 @@ int date_time_update_async(date_time_evt_handler_t evt_handler)
 	return 0;
 }
 
-int date_time_clear(void)
-{
-	time_aux.date_time_utc = 0;
-	time_aux.last_date_time_update = 0;
-	initial_valid_time = false;
```

Comment:
Unit tests fail as we cannot clear the state and especially `initial_valid_time` in `test_date_time_teardown()`.
Do we need to keep this for testing purposes?
@simensrostad

Reply:
This function has been reverted back